examples/nextjs/src/components/layout/header/header.js

```diff
@@ -9,7 +9,8 @@ function Header({ children }) {
     
 
     useEffect(() => {
-        setInsideEditor(isInsideEditor());
+        const uveContext = detectUVEContext()
+        setInsideEditor(isInsideEditor() && uveContext?.mode === EDITOR_MODE.EDIT);
```

Comment:
Why this double check? isn't `isInsideEditor` doing all what we need? and if not, why?

Reply:
In addition to knowing if we are inside the editor, we need to know if we are in preview or editing mode, since there are things that should only be shown in editing mode (such as the sortNavigation button)

We did not include this logic inside isInsideEditor() so as not to modify its original operation (we would have to pass flags to detect both things), so we put that logic inside detectUVEContext to use it as a starting point for future changes.

In the future, we could have both inside a single function.
`const { isInsideEditor, mode } = detectUVEContext()`
Or something similar